app/js/views/app_view.js

```diff
@@ -1,13 +1,35 @@
 define([
   'jquery',
   'backbone',
-  'text!templates/app_view.ejs'
-], function ($, Backbone, indexTemplate) {
+  'text!templates/app_view.ejs',
+  'collections/todos',
+  'models/todo'
+], function ($, Backbone, indexTemplate, TodoCollection, ToDo) {
   return Backbone.View.extend({
+    events: {
+      'keypress #todo-new-input': 'createNewOnEnter',
+      'click #todo-new-button': 'createNewOnClick'
+    },
+
+    ENTER_KEY: 13,
+
     el: this.$('#todo-app'),
 
-    currentDate: function () {
-      return new Date();
+    createNewOnEnter: function (e) {
+      if (e.which !== this.ENTER_KEY || !$('#todo-new-input').val().trim()) { return; }
```

Comment:
`!$('#todo-new-input').val().trim()` is duplicated with the one in `createNewOnClick` please reconsider this implementation.